rat king text fixes + others

yippeecore opened the pull request:
fixed a typo ('being' to 'bring'), and capitalized all mentions of Rat King. also changed the death results to say that the cat was killed by 'a' rat king rather than 'the' rat king, since the scarred results use 'a'. | originally reported here: https://github.com/ClanGenOfficial/clangen/issues/1818#issuecomment-1874356240

yippeecore committed:
rat king text fixes

fixed a typo ('being' to 'bring'), and capitalized all mentions of Rat King. also changed the death results to say that the cat was killed by 'a' rat king rather than 'the' rat king, since the scarred results use 'a'.

yippeecore committed:
gonetnr fix

fixes a minor issue where multiple cats are taken away by twolegs, when the flavor text mentions that only one cat  was taken.

yippeecore committed:
minor text edits

mostly stuck some ellipses' together (...) when they were originally spaced apart (. . .)

yippeecore committed:
handful of typo fixes

Ryos00 commented on resources/dicts/events/ceremonies/ceremony-master.json:
```diff
@@ -2269,7 +2269,7 @@
       "gloomy"
     ],
-    "(old_name) remains completely quiet as l_n names {PRONOUN/m_c/object} m_c after {PRONOUN/m_c/poss} r_h. Before anyone can congratulate {PRONOUN/m_c/object}, however, {PRONOUN/m_c/subject} {VERB/m_c/turn/turns} and walks away without a word to do {PRONOUN/m_c/poss} silent vigil."
+    "(old_name) remains completely quiet as l_n names {PRONOUN/m_c/object} m_c after {PRONOUN/m_c/poss} r_h. Before anyone can congratulate {PRONOUN/m_c/object}, however, {PRONOUN/m_c/subject} {VERB/m_c/turn/turns} and walk away without a word to do {PRONOUN/m_c/poss} silent vigil."
```
I think "walk" should actually be verb tagged as well

yippeecore replied:
done!

yippeecore committed:
Update ceremony-master.json

verb tagged 'walk' as per ryo's input - https://github.com/ClanGenOfficial/clangen/pull/2098#discussion_r1493818946

yippeecore committed:
Merge remote-tracking branch 'upstream/development' into development

yippeecore committed:
Merge branch 'development' of https://github.com/yippeecore/clangen into development

Thlumyn commented:
@Tiritirimatangi  Are we using themself or themselves as the object form of they? I personally think themselves is a bit more grammatically correct but it's more important that we're just consistent with the use throughout the rest of the game.

Tiritirimatangi commented:
> @Tiritirimatangi Are we using themself or themselves as the object form of they? I personally think themselves is a bit more grammatically correct but it's more important that we're just consistent with the use throughout the rest of the game.

@Thlumyn Themself is the current version - but obviously we can easily edit that in utility.py via pronoun tagging if we'd like to change that
